charts/fybrik/files/taxonomy/data_catalog_request.json

```diff
@@ -0,0 +1,17 @@
+{
+	"$schema": "http://json-schema.org/draft-04/schema#",
+	"title": "Data Catalog Request - GetDatasetDetails",
+	"type": "object",
+	"properties": {
+	  "assetID": {
+		"type": "string"
+	  },
+	  "operationType": {
+		"$ref": "taxonomy.json#/definitions/OperationType"
```

Comment:
No need to reference taxonomy.json here because it's not configurable at runtime.

Reply:
removed from base taxonomy and added the definition in data_catalog_request.json. Please review. Thanks.